.github/workflows/test-from-pypi.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11-dev']
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
there's one more that needs to be modified in this file https://github.com/enthought/traits/blob/2544fea11bb521153772baab4adaaa4b6c6469e8/.github/workflows/test-from-pypi.yml#L58

Reply:
Good catch; thanks. Fixed in b9bc5b96a28abe80897e0d25a9a44af5ce87b617.